deel-2-flexbox-layout/opdracht-1/styles/styles.css

```diff
@@ -1 +1,48 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+body {
+    height: 966px;
+
+    display: flex;
+    flex-direction: column;
+    /*flex-grow: 1;*/
+}
+
+header{
+    flex-grow: 0;
+    flex-shrink: 0;
+}
+
+main {
+    /*flex: 1 0 max-content;*/
+    /*height: 900px;*/
+
+    display: flex;
+    flex-direction: row;
+    flex-basis: auto;
+    flex-shrink: 0;
+    flex-grow: 1;
+    /*flex-direction: column;*/
+}
+
+article {
+    flex-basis: 0;
+    flex-grow: 2;
+    flex-shrink: 1;
+}
+
+nav {
+    flex-basis: 0;
+    flex-grow: 1;
+    flex-shrink: 0;
+}
+
+aside {
+    flex-basis: 0;
+    flex-grow: 1;
+    flex-shrink: 0;
+}
+
+footer {
+    flex-grow: 0;
```

Comment:
flex-basis needed here because it's in the column container which requires the height.